csharp/native/LinearSubTasking/Client/Program.cs

```diff
@@ -0,0 +1,199 @@
+                                   int    integer)
+    {
+      // Create gRPC channel to connect with ArmoniK control plane
+      var channel = GrpcChannelFactory.CreateChannel(new GrpcClient
+                                                     {
+                                                       Endpoint = endpoint,
+                                                     });
+
+      // Create client for task submission
+      var taskClient = new Tasks.TasksClient(channel);
+
+      // Create client for result creation
+      var resultClient = new Results.ResultsClient(channel);
+
+      // Create client for session creation
+      var sessionClient = new Sessions.SessionsClient(channel);
+
+      // Create client for events listening
+      var eventClient = new Events.EventsClient(channel);
+
+      // Default task options that will be used by each task if not overwritten when submitting tasks
+      var taskOptions = new TaskOptions
+                        {
+                          MaxDuration = Duration.FromTimeSpan(TimeSpan.FromHours(1)),
+                          MaxRetries  = 2,
+                          Priority    = 1,
+                          PartitionId = partition,
+                        };
+
+      // Request for session creation with default task options and allowed partitions for the session
+      var createSessionReply = sessionClient.CreateSession(new CreateSessionRequest
+                                                           {
+                                                             DefaultTaskOption = taskOptions,
+                                                             PartitionIds =
+                                                             {
+                                                               partition,
+                                                             },
+                                                           });
+
+      WriteLine($"sessionId: {createSessionReply.SessionId}");
+
+      // Create the result metadata and keep the id for task submission
+      var resultId = resultClient.CreateResultsMetaData(new CreateResultsMetaDataRequest
+                                                        {
+                                                          SessionId = createSessionReply.SessionId,
+                                                          Results =
+                                                          {
+                                                            new CreateResultsMetaDataRequest.Types.ResultCreate
+                                                            {
+                                                              Name = "Result",
+                                                            },
+                                                          },
+                                                        })
+                                 .Results.Single()
+                                 .ResultId;
```

Comment:
Can you print this resultId ? (mainly for easier debugging when there is an issue during task execution)